lib/sequenceserver.rb

```diff
@@ -11,7 +11,7 @@ module SequenceServer
   DEFAULT_CONFIG_FILE = '~/.sequenceserver.conf'.freeze
 
   # Constant for denoting the path ~/.sequenceserver
-  DOTDIR = File.expand_path('~/.sequenceserver').freeze
+  DOTDIR = File.expand_path('spec/dotdir').freeze
```

Comment:
Please be careful to not send code changes made for debugging as part of pull-requests.

Reply:
I forgot to not to add it.